src/catkin_pkg/package.py

```diff
@@ -473,7 +473,7 @@ def parse_package(path, warnings=None):
     try:
         return parse_package_string(xml, filename, warnings=warnings)
     except InvalidPackage as e:
-        e.args = ['Invalid package manifest "%s": %s' % (filename, e.message)]
+        e.args = ['Invalid package manifest "{}": {}'.format(filename, e)]
```

Comment:
Please revert the `{}` / `format` related changes and keep `%` since they fail the CI at the moment

Reply:
Changed to `{0}` and `{1}`, which are also python 2.6 compatible (there are more uses of `str.format()` in the same file already).